scripts/roscompile

```diff
@@ -1,15 +1,21 @@
 #!/usr/bin/python
 
+from __future__ import print_function
+
```

Comment:
I know its better form, but I'm stubbornly clinging to Python 2. Would you find it morally objectionable to remove this?